idf/overview.rst

```diff
@@ -0,0 +1,74 @@
+##############################
+Interim Data Facility Overview
+##############################
+
+Pre-operations activities may be carried out on Interim Data Facility (IDF) hosted on `Google Cloud Platform <https://cloud.google.com/>`_.
+The `Google Cloud documentation site <https://cloud.google.com/docs/overview>`_ provides an overview of its products and services.
+
+IDF is considered a production environment to support Data Preview releases.
+Development and Commissioning work continue on `LDF <../services/lsst-devl.rst>`_.
+
+
+Login credentials
+=================
+
+Rubin staff working on deploying and operating Data Production services receives a ``username@lsst.cloud`` identity.
+This IDF cloud identity is not an email and is independent from other Rubin accounts.
+Not everybody in the Data Production department needs this login; contact your pre-operations team leads if in doubt.
+
+2FA is required for the IDF identity.
+See :ref:`idf-internal-support` for lost password or login issues.
+
+
+Environments
+============
+
+On GCP, resources are organized into a base-level entity called "project".
+A project is linked to billing, permissions, and other settings.
+Projects are grouped in folders.
+We have six top-level folders:
+
+- ``Science Platform``
+- ``QServ``
+- ``Processing``
+- ``SQuaRE``
+- ``Shared Services``
+- ``Scratch``
+
+For each top-level folders except ``Shared Services`` and ``Scratch``, we have three sub-folders: ``Dev``, ``Integration``, and ``Production``.
+``Production`` is intended to host services used by end users.
+``Dev`` and ``Integration`` provide testing and staging environments where one can deploy near-production services.
+
+All environments on IDF are managed with `Terraform <https://www.terraform.io/>`_ infrastructure-as-code tool in the `idf_deploy <https://github.com/lsst/idf_deploy>`_ repo.
+See the documentations in `idf_deploy <https://github.com/lsst/idf_deploy>`_'s `run book <https://github.com/lsst/idf_deploy/tree/master/runbook>`_.
+GitHub Actions are used to automatically deploy GCP resources on IDF.
+All resources should be deployed via `idf_deploy <https://github.com/lsst/idf_deploy>`_ or layers on top of it, so operations can be repeated consistently.
```

Comment:
```suggestion
GitHub Actions are used to automatically deploy GCP resources on IDF in response to pull requests on the configuration files.
All resources should be deployed this way so that operations can be repeated consistently.
```